UserManagement/src/main/java/com/UserManagementBootstrapper.java

```diff
@@ -0,0 +1,21 @@
+package com;
```

Comment:
Kindly add logging into the next milestone.

Reply:
Thanks for pointing this out. 
will do it